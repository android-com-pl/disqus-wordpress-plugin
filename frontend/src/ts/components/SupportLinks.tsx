import * as React from 'react';

const SupportLinks = () => (
    <ul>
        <li>
            <a href='https://status.disqus.com/' target='_blank'>
                Disqus System Status
            </a>
        </li>
        <li>
            <a href='https://help.disqus.com/customer/en/portal/articles/472005' target='_blank'>
                WordPress Troubleshooting and F.A.Q.
            </a>
        </li>
        <li>
            <a href='https://disqus.com/home/channel/discussdisqus/' target='_blank'>
                Post Question on <em>Discuss Disqus</em>
            </a>
        </li>
        <li>
            <a href='https://github.com/disqus/disqus-wordpress-plugin' target='_blank'>
                Github Project (disqus-wordpress-plugin)
            </a>
        </li>
        <li>
            <a href='https://disqus.com/support/?article=contact_wordpress' target='_blank'>
                Contact Support
            </a>
        </li>
    </ul>
);

export default SupportLinks;
