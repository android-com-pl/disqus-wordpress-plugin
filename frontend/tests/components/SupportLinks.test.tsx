import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import SupportLinks from '../../src/ts/components/SupportLinks';

test('SupportLinks renders with correct links', () => {
    const component = TestRenderer.create(<SupportLinks />);

    expect(component).toMatchSnapshot();
});
