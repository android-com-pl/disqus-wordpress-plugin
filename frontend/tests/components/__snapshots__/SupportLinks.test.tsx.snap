// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportLinks renders with correct links 1`] = `
<ul>
  <li>
    <a
      href="https://status.disqus.com/"
      target="_blank"
    >
      Disqus System Status
    </a>
  </li>
  <li>
    <a
      href="https://github.com/disqus/disqus-wordpress-plugin"
      target="_blank"
    >
      Github Project (disqus-wordpress-plugin)
    </a>
  </li>
  <li>
    <a
      href="https://help.disqus.com/"
      target="_blank"
    >
      Documentation
    </a>
  </li>
  <li>
    <a
      href="https://disqus.com/support/?article=contact_wordpress"
      target="_blank"
    >
      Contact Support
    </a>
  </li>
</ul>
`;
